README.md

```diff
@@ -10,7 +10,8 @@ Supported Systems
 |:--------------:|:----------:|
 |     x86_64     |     Yes    |
 |      i686      |     Yes    |
-| arm (Exynos 5) |     Yes    |
+|     armv7l     |     Yes    |
+|     aarch64    |     Yes(use armv7l binaries)    |
```

Comment:
Second: although it does not break the table, I guess it is considered good practise to fix the spacing in the Markdown.